extension/src/test/e2e/extension.test.ts

```diff
@@ -130,32 +133,74 @@ describe('Experiments Table Webview', function () {
 })
 
 describe('Plots Webview', function () {
+  const webview = new PlotsWebview('plots')
+
   it('should load the plots webview with non-empty plots', async function () {
     this.timeout(60000)
-    const webview = new PlotsWebview('plots')
     const workbench = await browser.getWorkbench()
     await workbench.openCommandPrompt()
     await browser.keys([...'DVC: Show Plots', 'ArrowDown', 'Enter'])
 
     await waitForDvcToFinish()
-
     await webview.focus()
 
-    await browser.waitUntil(
-      async () => {
-        return (await webview.vegaVisualization$$.length) === 5
-      },
-      { timeout: 30000 }
-    )
+    await waitForAllPlotsToRender(webview, 5)
 
     const plots = await webview.vegaVisualization$$
+    for (const plot of plots) {
+      const plotNotEmpty = await webview.plotNotEmpty(plot)
+      expect(plotNotEmpty).toBe(true)
+    }
 
+    await webview.unfocus()
+    await closeAllEditors()
+  })
+
+  it('should create and delete a custom plot', async function () {
+    this.timeout(60000)
+    await createCustomPlot()
+    const workbench = await browser.getWorkbench()
+    await workbench.openCommandPrompt()
+    await workbench.executeCommand('DVC: Show Plots')
+
+    await waitForDvcToFinish()
+    await webview.focus()
+
+    await waitForAllPlotsToRender(webview, 6)
+
+    let plots = await webview.vegaVisualization$$
     for (const plot of plots) {
       const plotNotEmpty = await webview.plotNotEmpty(plot)
       expect(plotNotEmpty).toBe(true)
     }
 
     await webview.unfocus()
+    await closeAllEditors()
+
+    await deleteCustomPlot()
+    await workbench.executeCommand('DVC: Show Plots')
+
+    await waitForDvcToFinish()
+    await webview.focus()
+
+    await waitForAllPlotsToRender(webview, 5)
```

Comment:
[C] Seeing as you have made this helper you might want to make another one for checking the plots are not empty.

Reply:
Moved the code to a helper but ran in the [`expect-expect`](https://github.com/jest-community/eslint-plugin-jest/blob/v27.2.1/docs/rules/expect-expect.md#enforce-assertion-to-be-made-in-a-test-body-expect-expect) eslint warning. I disabled it with a comment for now but can revert the change if preferred!

Reply:
You can use a comment to update the expect function names: 
```
/* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["expect", "expectHeaders"] }] */
```